lib/langchain/llm/ollama.rb

```diff
@@ -64,7 +69,7 @@ def complete(prompt:, model: nil, **options)
     # @return [Langchain::LLM::OllamaResponse] Response object
     #
     def embed(text:, model: nil, **options)
```

Comment:
Could you please switch to something like what we're doing here: https://github.com/andreibondarev/langchainrb/blob/main/lib/langchain/llm/openai.rb#L49?

Also -- I'm trying to get rid of all of these `**options`, I think all of the options need to be spelled out otherwise it's confusing and impossible to know what can actually be passed in.